[2023-06-01T11:14:04.433+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-25T00:00:00+00:00 [queued]>
[2023-06-01T11:14:04.462+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-25T00:00:00+00:00 [queued]>
[2023-06-01T11:14:04.464+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:14:04.465+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:14:04.466+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:14:04.532+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2021-12-25 00:00:00+00:00
[2023-06-01T11:14:04.544+0000] {standard_task_runner.py:55} INFO - Started process 113 to run task
[2023-06-01T11:14:04.549+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'create_postgres_table', 'scheduled__2021-12-25T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmp7kawfzt5']
[2023-06-01T11:14:04.555+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask create_postgres_table
[2023-06-01T11:14:04.801+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table scheduled__2021-12-25T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:14:05.037+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-25T00:00:00+00:00
[2023-06-01T11:14:05.041+0000] {sql.py:253} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-06-01T11:14:05.060+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:14:05.885+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:14:15.908+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 331, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 144, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "host.docker.local" to address: Temporary failure in name resolution

[2023-06-01T11:14:15.928+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_operator_v03, task_id=create_postgres_table, execution_date=20211225T000000, start_date=20230601T111404, end_date=20230601T111415
[2023-06-01T11:14:15.948+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 14 for task create_postgres_table (could not translate host name "host.docker.local" to address: Temporary failure in name resolution
; 113)
[2023-06-01T11:14:15.998+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-06-01T11:14:16.032+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
